src/components/Farmhand/Farmhand.js

```diff
@@ -416,6 +419,10 @@ export default class Farmhand extends FarmhandReducers {
     return isOnline && room !== DEFAULT_ROOM
   }
 
+  get isForestUnlocked() {
+    return levelAchieved(this.state.experience) >= UNLOCK_FOREST_LEVEL
+  }
```

Comment:
While this is simple and straightforward, it might be better on balance to have this determined in `src/data/levels.js`. The centralized, linear nature of that file is very helpful for looking up level unlock rewards. It serves as self-documenting code, and I think it's worth sticking with that pattern unless it's infeasible to do so.

I'm thinking that we could have something like this in `levels.js`:

```js
levels[15] = {
  unlocksStageFocusType: stageFocusType.FOREST,
}
```

And then we'd need to update `getLevelEntitlements` to handle `unlocksStageFocusType` similarly to `unlocksTool`: https://github.com/jeremyckahn/farmhand/blob/4c394019ec5bdb9cfc1b117dcf64b6238b72ffce/src/utils/getLevelEntitlements.js#L36-L38

What do you think @lstebner?

Reply:
yeah, that makes sense to me. i was on the fence, but i'll go ahead and make the change!